client/lib/upgrades/actions/domain-management.js

```diff
@@ -126,17 +125,17 @@ function fetchDomains( siteId ) {
 		} );
 	}
 
+	const domains = DomainsStore.getBySite( siteId );
+	if ( domains.isFetching || domains.hasLoadedFromServer ) {
```

Comment:
What if we want to force a refresh of the list of domains? At first glance it looks like we shouldn't have a check on `hasLoadedFromServer ` but maybe I'm missing something.